.gitlab/generate-common.php

```diff
@@ -50,6 +50,9 @@ function unset_dd_runner_env_vars() {
       - runner_system_failure
       - scheduler_failure
       - api_failure
+      - script_failure
```

Comment:
Could this retry any script error and potentially mask real bugs that the tests uncover?

Reply:
From what I have seen so far, the tests are failing because of an external reason (a service like kafka is somehow not reachable for 1 particular test, or the language tests failing because of an public endpoint not being reachable, ect...)

Not really an error on our side most of the time, and when the error is then the tests are failing more than 2 times in a row